packages/ibc-utils/src/neutron.rs

```diff
@@ -38,19 +41,45 @@ pub fn ibc_send_message(
             .min_fee,
     );
     let total_fee = flatten_ntrn_ibc_fee(&ibc_fee);
-    let amount_minus_fee = amount
-        .checked_sub(total_fee.u128())
-        .ok_or_else(|| StdError::generic_err("Amount too low to pay for IBC transfer fees."))?;
+
+    // Sender's account balance for fee denom (NTRN)
+    let sender_ntrn_balance = deps.querier.query_balance(sender, NTRN_DENOM)?.amount;
+
+    let transfer_amount = if denom.to_string() == NTRN_DENOM {
+        if Uint128::from(amount) == sender_ntrn_balance {
+            // The full balance is being transferred .. deduct the fee from the transfer amount
+            Uint128::from(amount).checked_sub(total_fee)?
```

Comment:
I don't think this deduction is necessary, it gets "taken" by the ibc module automatically. Or is this meant to reserve the fees for an additional transfer in the future?

Reply:
AFAIK it needs to know the amount to send, but it can't send the full amount if the denom is NTRN, so we need to reduce the fee to get the amount we can send.